Cargo.toml

```diff
@@ -41,3 +50,6 @@ default-features = false
 default = ["std"]
 std = ["num-traits/std"]
 libm = ["num-traits/libm"]
+
+[dev-dependencies]
+rkyv = { version = "0.7", default-features = false, features = ["size_64"] }
```

Comment:
What is this for?

Note that `dev-dependencies` can't be optional, so this prevents running tests on older Rust.

Reply:
rkyv won't compile without either size_32 or size_64 features enabled. I figure out how to enable it through the test_all.sh script though :)